### PR TITLE
added client app.json

### PR DESCRIPTION
**Pull request by paribaker**
## What this does

Adds an app.json template for basic TN Apps, this is separate than the one we currently have which is set up for the bootstrapper's review apps

- [Jira ticket](Paste Jira link here) 

## Checklist
- [ ] Add script that changes the file name in place to app.json and removes the default one created for the bootstrapper pipeline

## Deploy Notes

n/a

## How to test

n/a

### CODE REVIEW

**Comment by oudeismetis** [comment]
Currently blocked as we can't test this while Heroku's connection to Github is disabled due to that security issue.

**Comment by oudeismetis** [comment]
Currently blocked as we can't test this while Heroku's connection to Github is disabled due to that security issue.

**Comment by paribaker** [comment]
@oudeismetis  is this still blocked? or even needed?

File: client-app.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      "size": "hobby"
+    }
+  },
+  "addons": [
+    "heroku-postgresql:hobby-dev"
+  ],
+  "environments": {
+    "review": {
+      "addons": [
+        "heroku-postgresql:hobby-dev"
```
Comment (oudeismetis) [inline client-app.json:27]: add "papertrail:choklad" in two spots here as well.

File: client-app.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Construct Capital LP",
```
Comment (oudeismetis) [inline client-app.json:2]: cookiecutter substitution

File: client-app.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "value": "staging"
+  },
+  "buildpacks": [
+    {
+      "url": "heroku/nodejs"
+    },
+    {
+      "url": "heroku/python"
+    }
+  ]
```
Comment (oudeismetis) [inline client-app.json:41]: `"scripts": { "postdeploy": "./scripts/db_setup.sh" }`

File: client-app.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "SECRET_KEY": {
+      "generator": "secret"
+    },
+    "NPM_CONFIG_PRODUCTION": false,
+    "ALLOWED_HOSTS": {
+      "value": ".herokuapp.com"
+    },
+    "DEBUG": {
+      "value": "True"
+    }
```
Comment (oudeismetis) [inline client-app.json:13]: there are a couple other env vars we can add here

**Comment by oudeismetis** [comment]
I don't normally like doing this, but there are a couple of small PRs like this that I'm going to merge into a single PR so we can get these things in.

**Comment by oudeismetis** [comment]
closing this PR in favor of https://github.com/thinknimble/tn-spa-bootstrapper/pull/129

**Pull request closed without merging**